python/tutorials/02-fused-softmax.py

```diff
@@ -221,9 +223,9 @@ def softmax(x):
         args={'M': 4096},  # values for function arguments not in `x_names` and `y_name`
     ))
 def benchmark(M, N, provider):
-    x = torch.randn(M, N, device='cuda', dtype=torch.float32)
-    stream = torch.cuda.Stream()
-    torch.cuda.set_stream(stream)
+    x = torch.randn(M, N, device=DEVICE, dtype=torch.float32)
+    stream = getattr(torch, DEVICE).Stream()
+    getattr(torch, DEVICE).set_stream(stream)
```

Comment:
does this work for other devices?

Reply:
I think so, I tested with XPU backend and the tutorials work without errors. As far as I remember, CI was launched here once and the 2nd tutorial was passed for HIP as well.